scripts/scil_validate_bids.py

```diff
@@ -188,106 +141,112 @@ def get_data(nSub, dwi, t1s, fs, associations, default_readout, clean):
-                    fmaps[index] = [x for xs in fmaps[index] for x in xs]
-            if "sbref" in associations[curr_dwi.filename].keys():
-                sbref[index] = associations[curr_dwi.filename]['sbref']
-                if len(sbref[index]) == 1 and isinstance(sbref[index][0], list):
-                    sbref[index] = [x for xs in sbref[index] for x in xs]
-
-            conversion = {"i": "x", "j": "y", "k": "z"}
-            dwi_metadata = get_metadata(curr_dwi)
-            if 'PhaseEncodingDirection' in dwi_metadata and index == 0:
-                dwi_PE = dwi_metadata['PhaseEncodingDirection']
-                dwi_PE = dwi_PE.replace(dwi_PE[0], conversion[dwi_PE[0]])
-                if len(dwi_PE) == 1:
-                    PE[index] = dwi_PE
-                    PE[index+1] = dwi_PE + '-'
-                else:
-                    PE[index] = dwi_PE
-                    PE[index+1] = dwi_PE[0]
-            elif clean:
-                return {}
-
-        # Find b0 for topup, take the first one
-        # Check fMAP
-        totalreadout = default_readout
-        fmaps = [fmap for fmap in fmaps if fmap != '']
-        if not fmaps:
-            if 'TotalReadoutTime' in dwi_metadata:
-                totalreadout = dwi_metadata['TotalReadoutTime']
-        else:
-            if isinstance(fmaps[0], list):
-                fmaps = [x for xs in fmaps for x in xs]
-
-            for nfmap in fmaps:
-                nfmap_metadata = get_metadata(nfmap)
-                if 'PhaseEncodingDirection' in nfmap_metadata:
-                    fmap_PE = nfmap_metadata['PhaseEncodingDirection']
-                    fmap_PE = fmap_PE.replace(fmap_PE[0], conversion[fmap_PE[0]])
-
-                    opposite_PE = PE.index(fmap_PE)
-                    if 'TotalReadoutTime' in dwi_metadata:
-                        if 'TotalReadoutTime' in nfmap_metadata:
-                            dwi_RT = dwi_metadata['TotalReadoutTime']
-                            fmap_RT = nfmap_metadata['TotalReadoutTime']
-                            if dwi_RT != fmap_RT and totalreadout == '':
-                                totalreadout = 'error_readout'
-                                topup_fmap[opposite_PE] = 'error_readout'
-                            elif dwi_RT == fmap_RT:
-                                topup_fmap[opposite_PE] = nfmap.path
-                                totalreadout = dwi_RT
-                    else:
-                        topup_fmap[opposite_PE] = nfmap.path
-                        totalreadout = default_readout
-
-        if sbref[index] != '' and len(sbref[index]) == 1:
-            topup_sbref[index] = sbref[index][0].path
-
-    if len(dwi) == 2:
-        if not any(s == '' for s in topup_sbref):
-            topup = topup_sbref
-        elif not any(s == '' for s in topup_fmap):
-            topup = topup_fmap
+
+    if len(dwis) == 2:
+        dwi_path[1] = dwis[1].path
+        bvec_path[1] = layout.get_bvec(dwis[1].path)
```

Comment:
When testing, here I get the bval/bvecs associated with the forward acquisition (not the reverse). Here is the json object generated for a sample subject :

````
{
        "DWIPhaseEncodingDir": "y",
        "TotalReadoutTime": 0.119974,
        "aparc_aseg": "",
        "bval": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_AP_dwi.bval",
        "bvec": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_AP_dwi.bvec",
        "dwi": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_AP_dwi.nii.gz",
        "rev_DWIPhaseEncodingDir": "y",
        "rev_bval": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_AP_dwi.bval",
        "rev_bvec": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_AP_dwi.bvec",
        "rev_dwi": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/dwi/sub-032215_ses-001-1_PA_dwi.nii.gz",
        "rev_topup": "",
        "run": 0,
        "session": "001",
        "subject": "032215",
        "t1": "/data/data/test_validate_bids/../test-tractoflow/sub-032215/ses-001-1/anat/sub-032215_ses-001-1_T1w.nii.gz",
        "topup": "",
        "wmparc": ""
}